colabs/wandb-artifacts/Model_Management_Guide.ipynb

```diff
@@ -44,14 +51,17 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
```

Comment:
Nit: The PR added your environment info to the notebooks metadata.

Reply:
Thank you, updated to remove these vscode-induced changes